src/30_Advanced/Amp-Carousel_with_Amp-List.html

```diff
@@ -0,0 +1,94 @@
+<!---{
+  "preview": "default"
+  }--->
+<!--
+  #### Introduction
+  AMP users might want to pull data from an endpoint to display in a carousel. This example incoporates a dynamic amp-carousel as well as showing how to not only rely on the images served from the endpoint
+   
+-->
+<!-- -->
+
+
+<!doctype html>
+<html ⚡>
+ <head>
+   <meta charset="utf-8">
+   <link rel="canonical" href="hello-world.html">
+   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+   <script async src="https://cdn.ampproject.org/v0.js"></script>
+   <!-- #### Setup -->
+  <!--
+    Import the `amp-list` and `amp-mustache` and `amp-carousel` and `amp-bind` components.
+  -->
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+   <!--
+    #### CSS
+  -->
+   <style amp-custom>
+       #alternate-images-carousel {
+    max-width:300px;
+    height:250px;
+    margin:0 auto;
+}
+    .carousel-wrapper {
+        height:260px;position:relative;
+    }
+    .hero-carousel-image {
+        max-width:225px;
+        height:auto;
+        width:100%;
+    }
+   </style>
+ </head>
+ <body>
+    <amp-state id="selected">
+        <script type="application/json">
+            {
+            "slide": 0
+            }
+        </script>
+    </amp-state>
+
+    <!-- This is a simplified version of our production release, 
+         some code is missing, but concepts will be explained -->
+    <div class="carousel-wrapper" >
+        <amp-list 
+            width="325" height="325" 
+            layout="fill" 
```

Comment:
suggestion: use a fixed-height layout which uses the same height as the carousel. This way you shouldn't need additional css.

Reply:
Will play with this and verify still works before my next commit